plugins/node/opentelemetry-instrumentation-bunyan/README.md

```diff
@@ -15,6 +15,10 @@ Compatible with OpenTelemetry JS API and SDK `1.0+`.
 npm install --save @opentelemetry/instrumentation-bunyan
 ```
 
+### Supported Versions
+
+- `^1.0.0`
```

Comment:
Should we change this to `1.8.12` since we only test that?

Reply:
I tested all versions in `^1.0.0` to validate the implementation, but only added versions that give us 95% of downloaded test coverage to TAV config. Versions outside of `^1.8.12` are seldom downloaded. See the stats:
<details>
 <summary>Stats</summary>

```
┌─────────┬──────────┬──────────────────────────┬──────────────┬───────────┬──────────┐
│ (index) │ version  │           time           │     tags     │ downloads │ ratio(%) │
├─────────┼──────────┼──────────────────────────┼──────────────┼───────────┼──────────┤
│    0    │ '1.8.15' │ 2021-01-08T23:26:28.125Z │ [ 'latest' ] │  777096   │   59.3   │
│    1    │ '1.8.12' │ 2017-08-02T23:58:58.696Z │      []      │  286208   │   21.8   │
│    2    │ '1.8.14' │ 2020-06-29T00:20:06.813Z │      []      │  144146   │    11    │
│    3    │ '1.8.13' │ 2020-06-24T04:27:55.281Z │      []      │   29975   │   2.3    │
│    4    │ '1.8.10' │ 2017-04-05T06:34:32.403Z │      []      │   10602   │   0.8    │
│    5    │ '1.3.4'  │ 2015-02-20T22:59:10.194Z │      []      │   8325    │   0.6    │
│    6    │ '1.8.1'  │ 2016-04-21T08:00:52.316Z │      []      │   8169    │   0.6    │
│    7    │ '1.5.1'  │ 2015-09-07T21:45:14.833Z │      []      │   5031    │   0.4    │
│    8    │ '1.0.1'  │ 2014-08-25T07:49:36.070Z │      []      │   2332    │   0.2    │
│    9    │ '1.8.5'  │ 2016-11-14T18:40:23.723Z │      []      │   1460    │   0.1    │
│   10    │ '1.8.8'  │ 2017-03-03T22:40:35.013Z │      []      │   1027    │   0.1    │
│   11    │ '1.8.0'  │ 2016-03-17T05:52:27.336Z │      []      │    904    │   0.1    │
│   12    │ '1.8.4'  │ 2016-10-25T18:24:07.753Z │      []      │    438    │    0     │
│   13    │ '1.1.3'  │ 2014-09-28T04:50:18.414Z │      []      │    375    │    0     │
│   14    │ '1.8.9'  │ 2017-03-21T05:26:11.895Z │      []      │    373    │    0     │
│   15    │ '1.7.1'  │ 2016-02-29T06:45:36.781Z │      []      │    352    │    0     │
│   16    │ '1.8.6'  │ 2017-03-03T18:55:55.423Z │      []      │    238    │    0     │
│   17    │ '1.4.0'  │ 2015-06-07T18:07:46.145Z │      []      │    223    │    0     │
│   18    │ '1.2.4'  │ 2015-01-16T06:22:58.580Z │      []      │    133    │    0     │
│   19    │ '1.0.0'  │ 2014-08-01T22:59:56.330Z │      []      │    133    │    0     │
│   20    │ '1.3.5'  │ 2015-04-13T05:18:46.303Z │      []      │    119    │    0     │
│   21    │ '1.8.2'  │ 2016-10-17T23:43:20.827Z │      []      │    80     │    0     │
│   22    │ '1.3.6'  │ 2015-06-02T05:13:17.059Z │      []      │    59     │    0     │
│   23    │ '1.8.7'  │ 2017-03-03T21:38:21.063Z │      []      │    58     │    0     │
│   24    │ '1.2.1'  │ 2014-10-26T04:07:20.692Z │      []      │    47     │    0     │
│   25    │ '1.3.3'  │ 2015-01-26T17:27:02.858Z │      []      │    40     │    0     │
│   26    │ '1.2.3'  │ 2014-11-18T05:47:22.069Z │      []      │    26     │    0     │
│   27    │ '1.8.3'  │ 2016-10-18T20:34:17.059Z │      []      │     7     │    0     │
│   28    │ '1.1.0'  │ 2014-09-22T04:18:55.436Z │      []      │     6     │    0     │
│   29    │ '1.8.11' │ 2017-08-02T20:23:19.830Z │      []      │     1     │    0     │
│   30    │ '1.7.0'  │ 2016-02-22T01:18:54.237Z │      []      │     1     │    0     │
│   31    │ '1.6.0'  │ 2016-02-11T06:36:32.420Z │      []      │     1     │    0     │
│   32    │ '1.5.0'  │ 2015-09-07T08:27:16.134Z │      []      │     1     │    0     │
│   33    │ '1.3.2'  │ 2015-01-19T07:32:26.714Z │      []      │     1     │    0     │
│   34    │ '1.3.1'  │ 2015-01-18T06:07:58.032Z │      []      │     1     │    0     │
│   35    │ '1.3.0'  │ 2015-01-17T21:37:13.164Z │      []      │     1     │    0     │
│   36    │ '1.2.2'  │ 2014-11-14T07:16:13.877Z │      []      │     1     │    0     │
│   37    │ '1.2.0'  │ 2014-10-17T21:35:40.227Z │      []      │     1     │    0     │
│   38    │ '1.1.2'  │ 2014-09-28T04:26:19.607Z │      []      │     1     │    0     │
│   39    │ '1.1.1'  │ 2014-09-28T04:08:56.763Z │      []      │     1     │    0     │
└─────────┴──────────┴──────────────────────────┴──────────────┴───────────┴──────────┘
Ratio of downloads through versions shown in the table: 97.5%
Total weekly downloads: 1,310,783
```

</details>